README.md

```diff
@@ -136,20 +145,39 @@ For Splunk Observability Cloud the following parameters are required:
   token.
 
 The following parameter is required for any of the destinations:
+
 - `clusterName`: arbitrary value that identifies your Kubernetes cluster. The value will be associated with every trace, metric and log as "k8s.cluster.name" attribute.
 
-To deploy the chart to send data to Splunk Observability Cloud run the following
-commands replacing the parameters above with their appropriate values.
+Run the following commands replacing the parameters above with their appropriate values.
+
+Add Helm repo
 
 ```bash
-$ helm repo add splunk-otel-collector-chart https://signalfx.github.io/splunk-otel-collector-chart
-$ helm install my-splunk-otel-collector --set="splunkObservability.realm=us0,splunkObservability.accessToken=xxxxxx,clusterName=my-cluster" splunk-otel-collector-chart/splunk-otel-collector
+helm repo add splunk-otel-collector-chart https://signalfx.github.io/splunk-otel-collector-chart
+```
+
+Sending data to Splunk Observability Cloud
+
+```bash
+helm install my-splunk-otel-collector --set="splunkObservability.realm=us0,splunkObservability.accessToken=xxxxxx,clusterName=my-cluster" splunk-otel-collector-chart/splunk-otel-collector
+```
+
+Sending data to Splunk Enterprise or Cloud
+
+```bash
+helm install my-splunk-otel-collector --set="splunkPlatform.endpoint=127.0.0.1:8088,splunkPlatform.token=xxxxxx,splunkPlatform.metrics_index=k8s-metrics,splunkPlatform.index=main,clusterName=my-cluster" splunk-otel-collector-chart/splunk-otel-collector
+```
+
+Sending data to both Splunk Observability Cloud and Splunk Enterprise or Cloud
+
+```bash
+helm install my-splunk-otel-collector --set="splunkPlatform.endpoint=127.0.0.1:8088,splunkPlatform.token=xxxxxx,splunkPlatform.metrics_index=k8s-metrics,splunkPlatform.index=main,splunkObservability.realm=us0,splunkObservability.accessToken=xxxxxx,clusterName=my-cluster"
 ```
 
```

Comment:
Probably we should mention native logs collection here. Something like:

Consider enabling [native OpenTelemetry logs collection](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#logs-collection) for better throughput instead of using default fluentd engine. Add the following part `--set=logsEngine=otel` to your installation command if you want to use native OpenTelemetry logs collection.